bioimage-io/UNet2DArabidopsisOvules.model/UNet2DArabidopsisOvules.model.yaml

```diff
@@ -0,0 +1,82 @@
+    doi: https://doi.org/10.1101/2020.01.17.910562
+authors:
+  - Adrian Wolny;@bioimage-io
+documentation: ../../README.md
+tags: [unet2d, pytorch, arabidopsis, ovuls, cell membrane, segmentation, plant tissue]
+license: MIT
+
+format_version: 0.1.0
+language: python
+framework: pytorch
+
+source: pytorch3dunet.unet3d.model.UNet2D
+optional_kwargs:
+  in_channels: 1
+  out_channels: 1
+  layer_order: gcr  # determines the order of operators in a single layer (crg - Conv3d+ReLU+GroupNorm)
+  f_maps: [32, 64, 128, 256]   # initial number of feature maps
+  num_groups: 8  # number of groups in the groupnorm
+  final_sigmoid: true  # apply element-wise nn.Sigmoid after the final 1x1x1 convolution, otherwise apply nn.Softmax
+  is_segmentation: true  # don't touch, use postprocessing instead
+  testing: false  # don't touch, use postprocessing instead
+
+weights:
+    - name: confocal_2D_unet_bce_dice_ds1x
+      source: https://oc.embl.de/index.php/s/61s67Mg5VQy7dh9/download?path=%2FArabidopsis-Ovules%2F2dunet_bce_dice_ds1x&files=best_checkpoint.pytorch
+      md5: 8667b25b8a078aac08f8c8d9dcadb15a
+      parent: null
+      description: "Trained with pixel size of (0.075, 0.075) microns"
+    - name: confocal_2D_unet_bce_dice_ds2x
+      source: https://oc.embl.de/index.php/s/61s67Mg5VQy7dh9/download?path=%2FArabidopsis-Ovules%2F2dunet_bce_dice_ds2x&files=best_checkpoint.pytorch
+      md5: 47ee0d24991e758eab65f87b2fc22de1
+      parent: null
+      description: "Trained with pixel size of (0.15, 0.15) microns (default)"
+    - name: confocal_2D_unet_bce_dice_ds3x
+      source: https://oc.embl.de/index.php/s/61s67Mg5VQy7dh9/download?path=%2FArabidopsis-Ovules%2F2dunet_bce_dice_ds3x&files=best_checkpoint.pytorch
+      md5: d6c9aaec8d0dd4153b89eab8d80472ee
+      parent: null
+      description: "Trained with pixel size of (0.225, 0.225) microns"
+
+# FIXME: is the test input/output necessary?
+#test_input: test_input.npz
+#test_output: test_output.npz
+covers: [raw.png, pred.png]
+
+inputs:
+  - name: raw
+    axes: bcyx
+    data_type: float32
+    data_range: [-inf, inf]
+    shape: [1, 1, 256, 256]
+
+outputs:
+  - name: cell_boundaries
+    axes: bcyx
+    data_type: float32
+    data_range: [0, 1]
+    halo: [0, 0, 16, 16]
+    shape:
+        reference_input: raw
+        scale: [1, 1, 1, 1]
+        offset: [0, 0, 0, 0]
+
+prediction:
+  preprocess:
+    - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/f71b8ac598267de88cd39e5495abd93dcda1d0a4/specs/transformations/EnsureTorch.transformation.yaml
+    - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/f71b8ac598267de88cd39e5495abd93dcda1d0a4/specs/transformations/Cast.transformation.yaml
+      kwargs: {dtype: float32}
+    - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/f71b8ac598267de88cd39e5495abd93dcda1d0a4/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
+      kwargs: {apply_to: [0]}
+  weights:
```

Comment:
Currently I specify the list of model weights. @FynnBe lemme know if this should rather be the 'default' entry instead of a list.
CC @constantinpape

Reply:
lets resolve this generally first and then continue here...
https://github.com/bioimage-io/configuration/pull/25#discussion_r442088052